zbuf/merger.go

```diff
@@ -71,24 +72,24 @@ func NewMerger(ctx context.Context, pullers []Puller, cmp expr.CompareFn) *Merge
 	return m
 }
 
-func MergeReadersByTsAsReader(ctx context.Context, readers []zio.Reader, order Order) (zio.Reader, error) {
+func MergeReadersByTsAsReader(ctx context.Context, readers []zio.Reader, o order.Which) (zio.Reader, error) {
```

Comment:
It's odd that `o` is named not for its type but for its type's package. Maybe rename `Which` to `Order` or `Ordering`?

Reply:
But it's which order of the two, a boolean.  order.Order is weird to me as is order.Ordering.  It's as to change in a subsequent PR as it is for me to change now and I'm not compelled with this proposal.  Maybe there's something better someone will come up and we can make the change later.

Reply:
The important thing is the shape of the order package is now correct, with a thing encompassing either order plus unknown/dontcare and another thing representing a strict order.

Reply:
Agreed!